src/main/java/core/basesyntax/db/FruitShopStorage.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax.db;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class FruitShopStorage {
+    private Map<String, Integer> fruitShopStorage;
+
+    public FruitShopStorage() {
+        this.fruitShopStorage = new HashMap<String, Integer>();
+    }
```

Comment:
You should have one database
```suggestion
    public FruitShopStorage() {
    }
```